src/legacy/core_plugins/kbn_vislib_vis_types/public/editors/gauge.tsx

```diff
@@ -0,0 +1,267 @@
+
+        <TextInputOption
+          disabled={!stateParams.gauge.labels.show}
+          label={i18n.translate('kbnVislibVisTypes.controls.gaugeOptions.subTextLabel', {
+            defaultMessage: 'Sub label',
+          })}
+          paramName="subText"
+          value={stateParams.gauge.style.subText}
+          setValue={(paramName, value) =>
+            setGaugeValue('style', { ...stateParams.gauge.style, [paramName]: value })
+          }
+        />
+
+        <SwitchOption
+          disabled={!stateParams.gauge.labels.show}
+          label={i18n.translate('kbnVislibVisTypes.controls.gaugeOptions.displayWarningsLabel', {
+            defaultMessage: 'Display warnings',
+          })}
+          tooltip={i18n.translate('kbnVislibVisTypes.controls.gaugeOptions.switchWarningsTipText', {
+            defaultMessage:
+              'Turns on/off warnings. When turned on, a warning will be shown if not all labels could be displayed.',
+          })}
+          paramName="isDisplayWarning"
+          value={stateParams.isDisplayWarning}
+          setValue={setValue}
+        />
+      </EuiPanel>
+
+      <EuiSpacer size="s" />
+
+      <EuiPanel paddingSize="s">
+        <EuiTitle size="xs">
+          <h2>
+            <FormattedMessage
+              id="kbnVislibVisTypes.controls.gaugeOptions.rangesTitle"
+              defaultMessage="Ranges"
+            />
+          </h2>
+        </EuiTitle>
+        <EuiSpacer size="s" />
+
+        <SwitchOption
+          disabled={stateParams.gauge.colorsRange.length < 2}
+          label={i18n.translate('kbnVislibVisTypes.controls.gaugeOptions.autoExtendRangeLabel', {
+            defaultMessage: 'Auto extend range',
+          })}
+          tooltip={i18n.translate('kbnVislibVisTypes.controls.gaugeOptions.extendRangeTipText', {
+            defaultMessage: 'Extends range to the maximum value in your data.',
+          })}
+          paramName="extendRange"
+          value={stateParams.gauge.extendRange}
+          setValue={setGaugeValue}
+        />
+        <EuiSpacer size="s" />
+
+        <RangesParamEditor
+          value={stateParams.gauge.colorsRange}
+          setValue={value => setGaugeValue('colorsRange', value)}
+        />
+
+        <EuiTitle size="xs">
+          <h2>
+            <FormattedMessage
+              id="kbnVislibVisTypes.controls.gaugeOptions.colorTitle"
+              defaultMessage="Color"
+            />
+          </h2>
+        </EuiTitle>
+        <EuiSpacer size="s" />
+
+        <SelectOption
+          disabled={stateParams.gauge.colorsRange.length < 2}
+          helpText={i18n.translate(
+            'kbnVislibVisTypes.controls.gaugeOptions.howToChangeColorsDescription',
+            {
+              defaultMessage: 'Note: colors can be changed in the legend.',
+            }
+          )}
+          label={i18n.translate('kbnVislibVisTypes.controls.gaugeOptions.colorSchemaLabel', {
+            defaultMessage: 'Schema',
```

Comment:
Prepend `Color` to this label
```suggestion
            defaultMessage: 'Color schema',
```